package.json

```diff
@@ -60,7 +60,7 @@
     "@lingui/macro": "^2.7.2",
     "babel-core": "^7.0.0-bridge.0",
     "husky": "^1.3.1",
-    "lint-staged": "^8.1.0",
+    "lint-staged": "^13.1.0",
```

Comment:
`Since v13.0.0 lint-staged no longer supports Node.js 12. Please upgrade your Node.js version to at least 14.13.1, or 16.0.0 onward.`